test/xpu_api/iterators/predef.iterators/move.iterators/move.iter.ops/move.iter.op.const/iter.pass.cpp

```diff
@@ -59,26 +57,25 @@ kernel_test()
                 ret_access[0] &= test(random_access_iterator<char*>(s));
                 ret_access[0] &= test(s);
 
-#if TEST_STD_VER > 14
                 {
                     constexpr const char* p = "123456789";
-                    constexpr s::move_iterator<const char*> it(p);
+                    constexpr dpl::move_iterator<const char*> it(p);
                     static_assert(it.base() == p);
                 }
-#endif
             });
         });
     }
     return ret;
 }
+#endif // TEST_DPCPP_BACKEND_PRESENT
 
 int
-main(int, char**)
+main()
 {
+#if TEST_DPCPP_BACKEND_PRESENT
     auto ret = kernel_test();
-    if (ret)
-        std::cout << "Pass" << std::endl;
-    else
-        std::cout << "Fail" << std::endl;
-    return 0;
+    EXPECT_TRUE(ret, "Wrong result of iterator check #2 in kernel_test()");
```

Comment:
=> "Wrong result of move_iterator check with value constructor in kernel_test()"

Reply:
Fixed